bevy_proto_backend/src/proto/prototypes.rs

```diff
@@ -48,6 +48,25 @@ impl<'w, T: Prototypical> PrototypesMut<'w, T> {
         handle
     }
 
+    /// Load the prototypes at the given path.
+    ///
+    /// This will also store strong handles to the prototypes in order to keep them loaded.
+    ///
+    /// To load without automatically storing the handles, try using [`AssetServer::load_folder`].
+    pub fn load_folder<P: Into<Cow<'static, str>>>(&mut self, path: P) -> Result<Vec<Handle<T>>, AssetServerError> {
+        let path = path.into();
+        let handles: Vec<_> = self.asset_server.load_folder(path.as_ref())?
+            .into_iter().map(|handle| handle.typed::<T>()).collect();
```

Comment:
I feel like we should leave these as `HandleUntyped` to align with `AssetServer::load_folder`. This also makes sense given that a folder may contain non-prototype assets.

Reply:
Fixed